test/primitive/rotation_gate.jl

```diff
@@ -58,3 +58,8 @@ end
         "θ in RotationGate is not real, got θ=$(g.theta), fallback to matrix-based method",
     ) isunitary(g) == false
 end
+
+@testset "occupied locs" begin
+    g = rot(put(5, 2=>X), 0.5)
```

Comment:
shouldn't this be written as `put(5, 2=>Rx(0.5))`? what's the actual use case for this?

Reply:
https://github.com/QuantumBFS/YaoPlots.jl/issues/28

Reply:
I am trying to fix the plotting.